qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -542,15 +542,101 @@ def from_operator(obj, atol=None, rtol=None):
 
     @staticmethod
     def from_list(obj):
-        """Construct from a list [(pauli_str, coeffs)]"""
+        """Construct from a list of Pauli strings and coefficients.
+
+        For example, the 5-qubit Hamiltonian
+
+        .. math::
+
+            H = X_0 Z_3 + 2 Y_1 Y_4
+
+        can be constructed as
+
+        .. code-block:: python
+
+            # via tuples and the full Pauli string
+            op = SparsePauliOp.from_list([("XIIZI", 1), ("IYIIY", 2)])
```

Comment:
The endianness is wrong in the example: `X_0 Z_3` over 4 qubits should produce `"ZIIX"`:
```
In [1]: from qiskit.quantum_info import Pauli
   ...: Pauli("ZIIX")[0]
Out[1]: Pauli('X')
```

(but the function is correct, since it doesn't use indices.)